core/src/ast/types.rs

```diff
@@ -793,104 +800,6 @@ impl TypeName {
     }
 }
 
-impl From<&syn::Type> for TypeName {
```

Comment:
q: Was this just unused?

Reply:
I think so. It duplicates `TypeName::from_ast`, which is what we actually use everywhere.